hybris/common/jb/linker.c

```diff
@@ -172,8 +172,28 @@ static struct link_map *r_debug_tail = 0;
 
 static pthread_mutex_t _r_debug_lock = PTHREAD_MUTEX_INITIALIZER;
 
+static int _is_android_debug_enabled() 
```

Comment:
lgtm but i think you could move the _is_android_debug_enabled function to hooks.c (where the linkers are loaded and pass the debug-enabled/disabled state via android_linker_init)
then we would avoid having the same code multiple times

Reply:
It should look better now!